index.html

```diff
@@ -641,6 +641,9 @@ <h2>DNS-Based Service Discovery</h2>
                         <dt><code>type</code></dt>
                         <dd>Type of the Thing Description, i.e. <code>Thing</code> or <code>Directory</code>.
                         If omitted, the type is assumed to be <code>Thing</code>.</dd>
+                        <dt><code>scheme</code></dt>
+                        <dd>"Scheme" part of URL.  One of the following values can be specified: <code>http</code>,<code>https</code>,<code>coap</code>, <code>coaps</code>, <code>coap+tcp</code> or <code>coaps+tcp</code>.
```

Comment:
Don't think we need quotes here.  Also, added RFC reference and made interpretations explicit (which clears up whether we are using TLS with CoAP when over TCP).
```suggestion
                        <dd>Scheme part of URL.  One of the following values can be specified, with the standard registered URI interpretations [[RFC7595]]: <code>http</code> (HTTP over TCP),<code>https</code> (HTTP over TLS/TCP), <code>coap</code> (CoAP over UDP), <code>coaps</code> CoAP over DTLS/UDP), <code>coap+tcp</code> (CoAP over TCP), or <code>coaps+tcp</code> (CoAP over TLS/TCP).
```